test/validators.js

```diff
@@ -2521,6 +2521,30 @@ describe('Validators', () => {
     });
   });
 
+  it('should validate rgb color strings', () => {
```

Comment:
Run `npm test`. It will generate files such as `validator.js`, `validator.min.js` and files in `lib/isRgbColor.js`. 
I think it might be needed. Not sure, but @profnandaa can confirm.

Reply:
okay, I wasn't sure where these files came from, so didn't commit them